contentcuration/contentcuration/static/js/edit_channel/exercise_creation/views.js

```diff
@@ -1422,7 +1425,14 @@ var HintModalView = BaseViews.BaseModalView.extend({
     show: function(){
         this.$(".hint_modal").modal({show: true});
     },
-    closing_hints:function(){
+    closing_hints:function(event){
+        if(!this.hint_editor.validate()) {
```

Comment:
To add one layer of QA, blank hints must be filled in or removed to close the hint modal

Reply:
Is there any reason that a this.hint_editor would be undefined?  If it ever was (for whatever reason), then this would throw an error.